src/components/Menu/menu.css

```diff
@@ -0,0 +1,32 @@
+.menu-page {
+  margin: 0 auto;
+  padding: 0;
+}
+
+.menu-page li:not(:last-child) {
+  margin-bottom: 30px;
+}
+
+.menu-page li {
+  list-style-type: none;
+}
+
+.menu-links {
+  display: block;
+  padding: 10px 0;
+  color: var(--secondary-text-color);
+  background-color: var(--primary-element-color);
+  border-radius: 30px;
+  text-align: center;
+}
+
+@media (min-width: 769px) {
+  .menu-page {
+    max-width: 100%;
+    width: 400px;
+  }
+
```

Comment:
why we have here "769" breakpoint? In 768px width this form will be stratched for 100% width

Reply:
resolved